### PR TITLE
added cursor-plus icon

### PR DESCRIPTION
**Pull request by johnmgriffith**
**Added**

cursor-plus

Screen Shot 20-05-15 02 54 00 PM

### CODE REVIEW

File: docs/keywords.json [hunk 1]
```diff
@@ -6069,5 +6069,16 @@
     ],
     "category":"Text",
     "release":"3.1.0"
+  }, 
+  "cursor-plus":{
+    "alias":[
+      "3.2.0",
```
Comment (paulcpederson) [inline docs/keywords.json:6075]: I could just make the release searchable in the docs site if that's why we've been adding the version as an alias...

Comment (johnmgriffith) [reply docs/keywords.json:6075]: That would work too.

**Comment by johnmgriffith** [comment]
Is there a way to recall a pull request? Nick is going to add to this request.

**Comment by paulcpederson** [comment]
you can just close it since it hasn't been merged.

**Comment by johnmgriffith** [comment]
Nick is going to add to this request with a few more icons. Once he's done, then we'll reopen this request

**Pull request closed without merging**